Firebase/Messaging/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2018-10-25 -- v3.2.1
+- Fixed an issue that message is failed to be delivered to the recipient's time zone. (#1946)
```

Comment:
nit: rephrase the middle `Fixed an issue where messages failed to be delivered to the recipient's time zone`